be more explicit that VC examples do not define anything (#405)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

# Additional Examples

> Important: The following examples are not normative and provided for

Author: ```suggestion
Important: The following examples are not normative and are provided for
```

illustrative purposes only. In particular, neither the structure of the claims

> nor the selection of selectively disclosable claims are normative.

Reviewer: ```suggestion
nor the selection of selectively disclosable claims is normative.
```

neither ... or ... is ... . I think

Author: is it is

## Example 4a - SD-JWT-based Verifiable Credentials (SD-JWT VC)

This example shows how the artifacts defined in this specification

> could hypothetically be used to express an

Reviewer: ```suggestion
could be used to express an
```

> This example shows how the artifacts defined in this specification
> could hypothetically be used to express an
> SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
> Person Identification Data (PID) defined in [@EUDIW.ARF].

Reviewer: ```suggestion
This is a non-normative example how the artifacts defined in this specification
could hypothetically be used to express an
SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
Person Identification Data (PID) defined in [@EUDIW.ARF].
```

Reviewer: ```suggestion
This example shows how the artifacts defined in this specification could be used to represent
SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
Person Identification Data (PID) defined in [@EUDIW.ARF] is used as a sample payload.
```

Reviewer: ok, so I think editors discussed to merge #407 first, and not cover in this PR?

Author: #407 was merged and addresses this part

## Example 4b - W3C Verifiable Credentials Data Model v2.0

> This hypothetical example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]

Reviewer: ```suggestion
This example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
```

> This hypothetical example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
> payload might be expressed using the artifacts defined in this specification.

Reviewer: ```suggestion
This is a non-normative example how the artifacts defined in this specification
could hypothetically be used to express
a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] payload.
```

Author: riffing on @Sakurann's suggestion 
```suggestion
This is non-normative example illustrates how the artifacts defined in this specification
could be used to express a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] payload.
```

* Do not disallow HMAC any longer.
* Editorial changes aimed at improved clarity
* Improve unlinkability considerations, mention that different KB keys must be used
* Be more explicit that the VCDM and SD-JWT VC examples are only illustrative and do not define anything
* Remove mention of unspecified key binding methods and the Enveloping SD-JWTs section

> * Update PID example

Author: I just happened to notice this was in the wrong draft revision when resolving conflicts merging in master

Reviewer 2: Thanks, I seem to be bad at putting changelog entries in the right place.

Author: Maybe because I indented -08 differently than the others? It's my fault...
